.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @eliasvaltteri @henrinie @michalksiezuk-nc @jererobles @vastamaki @nikolauska @biphobe @ViktorHegyvari @aashirjaved @Nissinen @PerttuSavolainen @lintula @iriekun
```

Comment:
Maybe add @terotil and the now-confirmed @katjajohanna :)

Reply:
Good point, added

Reply:
NOTE we have a limit of 15 reviewers, did we go past that already?

Reply:
it seems to be exactly 15 now. 😬

Reply:
I hope we will in the future start to split it up a bit so different people are responsible for different things. We could probably discuss about that at some point if more functionality is being added